ui/src/styles/components/top-bar/breadcrumb.scss

```diff
@@ -0,0 +1,35 @@
+:deep(.el-breadcrumb) {
```

Comment:
You can move on the `element-plus-overload.scss` since it a global layout for breadcrumb

Reply:
I made the change however I had to add !important on every property since element plus adds a file called breadcrumb.scss which takes precedence over everything since it is no more scoped to a component so selectors are less precise :'(